docker/services/logrotate/logrotate.d/simplified.conf

```diff
@@ -1,7 +1,6 @@
 /var/log/simplified/*.log {
     missingok
     daily
-    create 0700 root root
```

Comment:
Because we use `copytruncate` this line is actually ignored, so we might as well drop it.